### PR TITLE
New OSP rules

### PR DESCRIPTION
**Pull request by redhatrises**
Updated PR from #2894 as branch no longer exists.

### CODE REVIEW

**Comment by shawndwells** [comment]
`````
121/202 Test #122: validate-ssg-rhel-osp7-ds.xml ............................................***Failed    0.32 sec
File '/home/jenkins/workspace/scap-security-guide-pull-requests/label/rhel7/build/ssg-rhel-osp7-ds.xml' line 11532: Element '{http://checklists.nist.gov/xccdf/1.2}select', attribute 'idref': 'dangling reference to keystone_disable_user_account_days!' is not a valid value of the atomic type 'xs:NCName'.
File '/home/jenkins/workspace/scap-security-guide-pull-requests/label/rhel7/build/ssg-rhel-osp7-ds.xml' line 11532: Element '{http://checklists.nist.gov/xccdf/1.2}select', attribute 'idref': Warning: No precomputed value available, the value was either invalid or something strange happend.
OpenSCAP Error: Invalid SCAP Source Datastream (1.2) content in /home/jenkins/workspace/scap-security-guide-pull-requests/label/rhel7/build/ssg-rhel-osp7-ds.xml. [oscap_source.c:342]

`````

File: rhel-osp7/profiles/stig-openstack.profile [hunk 1]
```diff
@@ -21,12 +21,25 @@ selections:
...
+
     - keystone_file_ownership
     - keystone_file_perms
     - keystone_use_ssl
     - keystone_algorithm_hashing
     - keystone_max_request_body_size
     - keystone_disable_admin_token
+    - keystone_lockout_failure_attempts
+    - keystone_lockout_duration
+    - keystone_disable_user_account_days
```
Comment (shawndwells) [inline rhel-osp7/profiles/stig-openstack.profile]: should be ``keystone_disable_user_account_days_inactive``

Comment (redhatrises) [reply rhel-osp7/profiles/stig-openstack.profile]: @shawndwells fixed

**Comment by shawndwells** [comment]
+cc @noslzzp: some sample OSP content merged.

Thanks, @redhatrises

**Pull request merged**